src/js/components/Pagination/StyledPageControl.js

```diff
@@ -4,9 +4,6 @@ import { Button } from '../Button';
 
 const BUTTON_STATES = ['normal', 'active', 'disabled', 'hover'];
 const defaultStyles = {
-  border: {
-    radius: '4px',
-  },
   pad: '4px',
```

Comment:
Can this be defined in the theme anywhere and remove need for defaultStyles inline?

Reply:
Yes. Will adjust.

Reply:
Moved to theme and documented.